.agents/journal/sentinel.md

```
@@ -21,3 +21,8 @@ permissions at creation time, and they default to the user's `umask`. For files
 explicit permission management is required.
 **Prevention:** Always apply restrictive permissions (e.g., `0600` on Unix) to files generated by the
 tool that are known to contain user-provided credentials or sensitive environment variables.
+
+## 2025-05-17 - Information Disclosure via Symlink Following in Skill Installation
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)